tips/TIP-0046/tip-0046.md

```diff
@@ -0,0 +1,858 @@
+- Let `Block ID` be the concatenation of `ID` and `Serialized Slot Index`.
+
+The string format of the Block ID is the hexadecimal encoding of the `Block ID` with a `0x` prefix.
+
+## Basic Block
+
+### Schema
+
+A _Basic Block_ is the name for a _Block_ with its body set to a _Basic Block Body_, which is defined as:
+
+<details>
+  <summary>Basic Block Body</summary>
+  <blockquote>The basic block body.</blockquote>
+</details>
+<table>
+  <tr>
+    <td>
+      <b>Name</b>
+    </td>
+    <td>
+      <b>Type</b>
+    </td>
+    <td>
+      <b>Description</b>
+    </td>
+  </tr>
+  <tr>
+    <td>Block Body Type</td>
+    <td>uint8</td>
+    <td>Set to <strong>value 0</strong> to denote a <i>Basic Block Body</i>.</td>
+  </tr>
+  <tr>
+    <td>Strong Parents Count</td>
+    <td>uint8</td>
+    <td>The number of blocks following, which are strongly directly approved.</td>
+  </tr>
+  <tr>
+    <td valign="top">Strong Parents <code>anyOf</code></td>
+    <td colspan="2">
+      <details>
+        <summary>Parent</summary>
+        <blockquote>A reference to a block. Defined in <a href='../TIP-0046/tip-0046.md#parent'>TIP-46 (Parent)</a>.</blockquote>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Weak Parents Count</td>
+    <td>uint8</td>
+    <td>The number of blocks following, which are weakly directly approved.</td>
+  </tr>
+  <tr>
+    <td valign="top">Weak Parents <code>optAnyOf</code></td>
+    <td colspan="2">
+      <details>
+        <summary>Parent</summary>
+        <blockquote>A reference to a block. Defined in <a href='../TIP-0046/tip-0046.md#parent'>TIP-46 (Parent)</a>.</blockquote>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Shallow Like Parents Count</td>
+    <td>uint8</td>
+    <td>The number of blocks following, which are directly referenced to adjust opinion.</td>
+  </tr>
+  <tr>
+    <td valign="top">Shallow Like Parents <code>optAnyOf</code></td>
+    <td colspan="2">
+      <details>
+        <summary>Parent</summary>
+        <blockquote>A reference to a block. Defined in <a href='../TIP-0046/tip-0046.md#parent'>TIP-46 (Parent)</a>.</blockquote>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Payload Length</td>
+    <td>uint32</td>
+    <td>The length of the following payload in bytes. A length of 0 means no payload will be attached.</td>
+  </tr>
+  <tr>
+    <td valign="top">Payload <code>optOneOf</code></td>
+    <td colspan="2">
+      <details>
+        <summary>Signed Transaction</summary>
+        <blockquote>A transaction with its unlocks. Defined in <a href='../TIP-0045/tip-0045.md#signed-transaction'>TIP-45 (Signed Transaction)</a>.</blockquote>
+      </details>
+      <details>
+        <summary>Tagged Data</summary>
+        <blockquote>Data with an optional tag. It is defined in <a href='../TIP-0023/tip-0023.md#specification'>TIP-23 (Specification)</a> with the exception of the type value, which must be set to <b>value 0</b> to be compatible with this TIP.</blockquote>
```

Comment:
Should maybe point to a revised TaggedData TIP since the type byte changed ?

Reply:
I was trying to avoid having to redefine the entire TIP where only one character is different, but it would be cleaner to redefine I suppose.